server/lease/lessor.go

```diff
@@ -198,6 +201,7 @@ type LessorConfig struct {
 	CheckpointInterval         time.Duration
 	ExpiredLeasesRetryInterval time.Duration
 	CheckpointPersist          bool
+	LeaseRevokeRate            int
```

Comment:
Usually an exported field can be configured by users, but this field is just used internally for now.  We can export it and allow users to change its value if needed in future.
```suggestion
	leaseRevokeRate            int
```

Reply:
+1

Reply:
@ahrtr thanks for the feedback. Updated to make `leaseRevokeRate` a private field